cv32e40p/sim/tools/vsim/exclusion/README

```diff
@@ -0,0 +1,47 @@
+This directory contains waiver files for cv32e40p v2 verification.
```

Comment:
If this file is named `README.md`, then GitHub will automatically render it into html when you view it with your browser.  (Obviously, this is just FYI - no need to gate the merge on this trivial item.)